app/assets/stylesheets/components/_banner.scss

```diff
@@ -0,0 +1,5 @@
+
+// banner {
```

Comment:
I think only line 2 has been commented out and the rest remains active? So either all the lines would have to be commented out or none. But maybe I am also seeing it wrong because of the way it is diplayed here. 😄

Reply:
ah. Yes. it's actually not a problem. Because this component isn't called anywhere. It's a reminder of where to create tit when home page will be implemented

Reply:
👍  ok cool.